ide/docker.api/src/org/netbeans/modules/docker/IgnorePattern.java

```diff
@@ -108,7 +108,7 @@ static String preprocess(String pattern, char separator) {
         }
 
         ret = ret.replaceAll("^(" + Pattern.quote(sep) + "\\.\\.)+(" + Pattern.quote(sep) +")?", Matcher.quoteReplacement(sep))
-                .replaceAll("/", Matcher.quoteReplacement(sep));
+                .replace("/", Matcher.quoteReplacement(sep));
```

Comment:
this might be wrong since `quoteReplacement` will have a different effect when passed to `replace` instead of `replaceAll`

Reply:
great